packages/openneuro-app/src/scripts/dataset/files/file-viewer-type.jsx

```diff
@@ -19,7 +19,7 @@ const FileViewerType = ({ path, url, data }) => {
     path.endsWith('.txt')
   ) {
     return <FileViewerText data={data} />
-  } else if (path.endsWith('.nii.gz') || path.endsWith('.nii')) {
+  } else if (path.endsWith('.nii.gz') || path.endsWith('.nii') || path.endswith('.mgh') || path.endswith('.mgz') {
```

Comment:
```suggestion
  } else if (path.endsWith('.nii.gz') || path.endsWith('.nii') || path.endsWith('.mgh') || path.endsWith('.mgz')) {
```
Missing close paren and camel casing for endsWith.